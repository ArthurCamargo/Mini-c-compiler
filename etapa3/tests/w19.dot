digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n1 -> n4;
n2;
n2 [label="a"];
n3;
n3 -> n5;
n3 -> n6;
n5;
n5 [label="a"];
n6;
n6 [label="b"];
n3 [label="="];
n4;
n4 -> n7;
n4 -> n8;
n7;
n7 [label="a"];
n8;
n8 [label="3"];
n4 [label="="];
n1 [label="if"];
n0 [label="f"];

}