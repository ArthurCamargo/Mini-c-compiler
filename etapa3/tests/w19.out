0x56179875f350
0x56179875f350, 0x56179875f750
0x56179875f750
0x56179875f750, 0x56179875f4c0
0x56179875f750, 0x56179875f5f0
0x56179875f750, 0x56179875f700
0x56179875f4c0
0x56179875f4c0 [label="a"];
0x56179875f5f0
0x56179875f5f0, 0x56179875f530
0x56179875f5f0, 0x56179875f5a0
0x56179875f530
0x56179875f530 [label="a"];
0x56179875f5a0
0x56179875f5a0 [label="b"];
0x56179875f5f0 [label="="];
0x56179875f700
0x56179875f700, 0x56179875f660
0x56179875f700, 0x56179875f6b0
0x56179875f660
0x56179875f660 [label="a"];
0x56179875f6b0
0x56179875f6b0 [label="3"];
0x56179875f700 [label="="];
0x56179875f750 [label="if"];
0x56179875f350 [label="f"];
